src/main/java/com/elixr/poc/rest/response/PurchaseResponse.java

```diff
@@ -0,0 +1,33 @@
+package com.elixr.poc.rest.response;
+
+import lombok.Builder;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * This Class holds the response attributes for purchase related end points.
+ */
+
+@Data
+public class PurchaseResponse extends ErrorResponse {
+
+    @Id
```

Comment:
Could you please let me know why Id is added here?

Reply:
And I believe this is a data object?

Reply:
@ID is not required here, so i removed this.